### PR TITLE
Fixes* surgery steps to fix FBP faces

### LINKED ISSUE #803: Prosthetic Faces not reparable/replacable
Didn't see anything in the surgery\robotics code about this, but it seems that if a synthetic head melts/explodes it's impossible repair the head to the point where "Unknown (as Robot McNoFace)" stops happening, which would indicate a proper repair. Steps tried included repairing the head internals, repairing the head externals, targeting the face during both of the above, and fully replacing the head.

### PR DESCRIPTION
**Pull request by Datraen**
Should fix #803.

### CODE REVIEW

**Comment by Zuhayr** [comment]
https://github.com/PolarisSS13/Polaris/blob/master/code/modules/surgery/robotics.dm#L150

If this isn't working then that specific surgery step is bugged, adding a whole set of new ones isn't a good solution.

File: code/modules/surgery/robotics.dm [hunk 1]
```diff
@@ -426,4 +426,137 @@
...
+		var/obj/item/organ/I = target.internal_organs_by_name[eyes]
+		if(isnull(I))
+			return
+		user.visible_message("<span class='warning'>[user]'s hand slips, hitting [target]'s optical sensors!</span>", \
+		"<span class='warning'>Your hand slips, hitting [target]'s optical sensors!</span>")
+		I.damage = round(rand(4, 11))
+
+/datum/surgery_step/robotics/finish_face
+	allowed_tools = list(
+		/obj/item/weapon/weldingtool = 35,
```
Comment (Anewbe) [inline code/modules/surgery/robotics.dm]: Maybe a tool other than the welder?

**Comment by Datraen** [comment]
@Zuhayr, I thought it was strange that you had left out these sections in your rewrite, but it makes sense now that you've pointed out your consolidation. I had missed the implementation in the fixing of brute and burn.

**Comment by Datraen** [comment]
Worked fine with burn, brute damage to head in non-synthetic tests.

**Comment by Datraen** [comment]
To clarify the testing that was done:
I mashed an FBP's face in to the point of disfiguring, and repaired it.
I then melted an FBP's face with an e-gun, and repaired it.

**Pull request merged**